doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -30,10 +57,51 @@
 
+The distribution and version of Kubernetes used must be listed in the [Kubernetes Distributions and Platforms document](https://docs.google.com/spreadsheets/d/1LxSqBzjOxfGx3cmtZ4EbB_BGCxT_wlxW_xgHVVa23es/edit#gid=0) which lists all those that are Certified as being conformant with the CNCF Kubernetes specification.
+
+This Reference Architecture also specifies:
+
+- Master nodes must run the following Kubernetes control plane services:
+    - kube-apiserver
+    - kube-scheduler
+    - kube-controller-manager
+- Master nodes can also run the etcd service and host the etcd database, however etcd can also be hosted on separate nodes
+- In order to support `req.gen.rsl.01`, `req.gen.rsl.02` and `req.gen.avl.01` a Reference Implementation must:
+    - Consist of either three, five or seven nodes running the etcd service (can be colocated on the master nodes, or can run on separate nodes, but not on worker nodes)
+    - At least one master node per availability zone or fault domain to ensure the high availability and resilience of Kubernetes control plane services
+    - At least one worker node per availability zone or fault domain to ensure the high availability and resilience of workloads managed by Kubernetes
+- Master node services, including etcd, and worker node services (e.g. consumer workloads) must be kept separate - i.e. there must be at least one master node, and at least one worker node
+- Workloads must ***not*** rely on the availability of the master nodes for the successful execution of their functionality (i.e. loss of the master nodes may affect non-functional behaviours such as healing and scaling, but components that are already running will continue to do so without issue)
+- The following kubelet features must be enabled
+    - CPU Manager
+    - Device Plugin
+    - Topology Manager
+
+All kubelet features can be enabled/disabled by using the `feature-gates:` section in the kubelet config file.  e.g.
+```
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+feature-gates:
+  CPUManager: true|false (BETA - default=true)
+  DevicePlugins: true|false (BETA - default=true)
+  TopologyManager: true|false (ALPHA - default=false)
+```
+
 <a name="4.4"></a>
 ## 4.4 Container runtimes
 
-> This chapter should describe which container runtimes are part of the Reference Architecture.
+In order to support `req.inf.com.03`, the chosen runtime must be compliant with the [Kubernetes Container Runtime Interface (CRI)](https://kubernetes.io/blog/2016/12/container-runtime-interface-cri-in-kubernetes/) and the [Open Container Initiative (OCI) runtime spec](https://github.com/opencontainers/runtime-spec). Examples of container runtimes that are compliant with these specification are (note this is not a complete list and in no particular order):
+- container-d (with CRI plugin enabled, which it is by default)
+- rkt
```

Comment:
is rkt still a thing? also it does not support OCI format, therefore incompatible with the previous statement

Reply:
rkt was [archived by CNCF](https://www.cncf.io/archived-projects/)

Reply:
Will remove - thanks.  Can this be resolved?

Reply:
sure, thanks